src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -51,8 +51,8 @@ function getValueU64(ptr: number) {
 }
 
 export const monoPlatform: Platform = {
-  load: function load(options: Partial<WebAssemblyStartOptions>) {
-    return createRuntimeInstance(options);
+  load: function load(options: Partial<WebAssemblyStartOptions>, onConfigLoaded?: (loadedConfig: MonoConfig) => void) {
```

Comment:
I think that the callback could be added to `WebAssemblyStartOptions`
Or is that a public type ?

Reply:
Yes, `WebAssemblyStartOptions` is a public type